File: Fórmula_de_Bhaskara.py
import math 
A,B,C = input().split()
A = float(A)
B = float(B)
C = float(C)

aux =  (math.pow(B, 2) - (4 * A * C))
if aux<0 or A==0:
    print("Impossivel calcular")
else: 
    D = math.sqrt((math.pow(B, 2) - (4 * A * C)))
    R1 = ((-B + D) / (2 * A))
    R2 = ((-B - D) / (2 * A))
    print("R1 = {:.5f}".format(R1))
    print("R2 = {:.5f}".format(R2))

File: Fórmula_de_Bhaskara.js
const fs = require("fs");
const r = fs.readFileSync(0, "utf8");
let line = r.split(" ");
let a = parseFloat(line[0]);
let b = parseFloat(line[1]);
let c = parseFloat(line[2]);
let delta = Math.pow(b, 2) - 4 * a * c;
if (delta < 0 || a === 0) {
  console.log("Impossivel calcular");
} else {
  console.log(`R1 = ${((-b + Math.sqrt(delta)) / (2 * a)).toFixed(5)}`);
  console.log(`R2 = ${((-b - Math.sqrt(delta)) / (2 * a)).toFixed(5)}`);
}
